tests/test.py

```diff
@@ -266,6 +266,40 @@ def test_incorrect_key(self):
         self.assertRaises(InvalidKeyException, loader.user_id, "] [")
         self.assertRaises(InvalidKeyException, loader.map_id, "9d0a8fec2fe3f778334df6bdc60b113c")
 
+class TestEquality(CGTestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.cg = Circleguard(KEY)
+
+        cls.user = User(2757689, num=2) # toy, #1=sidetracked day, #2=View of The River Styx
+        cls.user1 = User(2757689, num=2, cache=False)
+        cls.user2 = User(2757689, num=2, mods=Mod.HT)
+        cls.user3 = User(2757689, num=1)
+
+        cls.map = Map(1754777, num=4) #sidetracked day: umbre, karthy, -duckleader-, toy
+        cls.map1 = Map(1754777, num=4, cache=False)
+        cls.map2 = Map(1754777, num=4, mods=Mod.HD)
+        cls.map3 = Map(1754777, num=2)
+
+        cls.r = ReplayMap(1754777, 2766034) # umbre +HDHR on sidetracked day
+
+        cls.cg.load_info(cls.user)
+        cls.cg.load_info(cls.user1)
+        cls.cg.load_info(cls.user2)
+
+        cls.cg.load_info(cls.map)
+        cls.cg.load_info(cls.map1)
+        cls.cg.load_info(cls.map2)
+
+    def test_identical_equality(self):
+        self.assertEqual(self.user, self.user1)
+        self.assertNotEqual(self.user, self.user2)
+        self.assertNotEqual(self.user, self.user3)
+
+        self.assertEqual(self.map, self.map1)
+        self.assertNotEqual(self.map, self.map2)
+        self.assertNotEqual(self.map, self.map3)
```

Comment:
I would like to have them split in separate tests
```python
def test_user_equality(self):
    self.assertEqual(self.user, self.user1)

def test_user_inequality1(self):
    self.assertNotEqual(self.user, self.user2)

def test_user_inequality2(self):
    self.assertNotEqual(self.user, self.user3)
```
bit inefficient but having them separate tests is useful since it instantly shows what is broken, not only that something is broken.

Reply:
That's a bit too wordy for me, and not really how the rest of the tests are set up. I think they should be split into user and map, but nothing beyond that